packages/button/src/__demo__/examples/iconOnly.js

```diff
@@ -29,25 +29,27 @@ export default {
   title: 'Icon-only Buttons',
   description:
     'Buttons that contain only Icons can use either `iconStart` or `iconEnd` props and must have an `aria-label` provided.',
-  scope: { Button, IconQueue, DemoLayout },
+  scope: { Button, IconCloud, DemoLayout },
   source: `() => {
-  const icon = <IconQueue color="currentColor" />;
+  const icon = <IconCloud />;
 
   return <DemoLayout>
     {/* Icon as prop; no text. aria-label applied to Button. */}
-    <Button iconStart={icon} aria-label="Add new" />
+    <Button iconStart={icon} aria-label="Cloud" />
     {/* Icon as child; no text. aria-label applied to Button. Note that the size of the icon must be provided in this case. */}
```

Comment:
This example no longer looks correct because of the new padding styles, and I didn't see an easy/straightforward way to fix it. Should we just remove it?